global.json

```diff
@@ -1,9 +1,9 @@
 {
   "tools": {
-    "dotnet": "2.1.403"
+    "dotnet": "3.0.100-preview3-010395"
```

Comment:
Is this a publicly released preview?  I believe whatever you list here will be the minimum required by contributors when working in VS as well.